Apply linter suggestions to #65137

malfet opened the pull request:
Summary: Just run linter on the change and apply all suggestions

Test Plan: N/A

Differential Revision: D31102960

driazati commented:
any thoughts on https://github.com/pytorch/pytorch/pull/65249? It’d save some manual work in cases like these

malfet committed:
Apply linter suggestions to #65137 (#65459)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/65459

Just run linter on the change and apply all suggestions

Test Plan: N/A

Reviewed By: seemethere

Differential Revision: D31102960

fbshipit-source-id: 8487b39df880e2b8f1e3334d23e308db522fea0f
